### PR TITLE
[test](ut) added testcases for show analyze statement

### PR DESCRIPTION
**Pull request by Nitin-Kashyap**
## Proposed changes

added UT cases for ShowAnalyzeStmt.

### CODE REVIEW

**Comment by yiguolei** [comment]
run buildall

**Comment by Nitin-Kashyap** [comment]
run buildall

**Review by morningman — DISMISSED** [review]
LGTM

File: fe/fe-core/src/test/java/org/apache/doris/analysis/ShowAnalyzeStmtTest.java [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+import org.apache.doris.catalog.ScalarType;
+import org.apache.doris.common.AnalysisException;
+import org.apache.doris.common.ErrorCode;
+import org.apache.doris.common.ErrorReport;
+import org.apache.doris.common.UserException;
+import org.apache.doris.qe.ConnectContext;
+import org.apache.doris.qe.SessionVariable;
+import org.apache.doris.qe.ShowResultSetMetaData;
+
+import mockit.Mock;
```
Comment (morningman) [inline fe/fe-core/src/test/java/org/apache/doris/analysis/ShowAnalyzeStmtTest.java:30]: The import order is wrong.
http://43.132.222.7:8111/buildConfiguration/Doris_DorisCompile_Compile/325463?

you can refer to 
https://doris.apache.org/community/developer-guide/java-format-code

**Review by Vallishp — APPROVED** [review]
LGTM

File: fe/fe-core/src/test/java/org/apache/doris/analysis/ShowAnalyzeStmtTest.java (hunk 1 above)
Comment (Nitin-Kashyap) [reply fe/fe-core/src/test/java/org/apache/doris/analysis/ShowAnalyzeStmtTest.java:30]: fixed

**Comment by Nitin-Kashyap** [comment]
run buildall

**Review by Vallishp — APPROVED** [review]
LGTM

**Comment by Nitin-Kashyap** [comment]
run buildall

**Comment by Nitin-Kashyap** [comment]
run buildall

**Pull request closed without merging**